codechecker_common/util.py

```diff
@@ -33,6 +33,19 @@ def arg_match(options, args):
     return matched_args
 
 
+def get_linef(fp, line_no):
+    """'fp' should be (readable) file object.
+    Return the line content at line_no or an empty line
+    if there is less lines than line_no.
+    """
+    fp.seek(0)
+    for line in fp:
+        line_no -= 1
+        if line_no == 0:
+            return line
+    return ''
```

Comment:
To get the number of lines in a source file like (len(lines)) probably it has to iterate over the lines, which would be done always even if the first line should be returned, I do not know if it would have some performance benefit, we should measure it.